actions/contacts/search.go

```diff
@@ -15,7 +17,7 @@ import (
 // @Tags		Contact
 // @Produce		json
 // @Param		SearchContacts body SearchContacts false "Supports targeted resource searches with filters and metadata, plus options for pagination and sorting to streamline data exploration and analysis"
-// @Success		200 {object} []models.Contact "List of contacts"
+// @Success		200 {object} common.SearchContactsResponse "List of contacts"
```

Comment:
After moving it to `models` I suppose it should be here `models.SearchContactsResponse`.

Reply:
Done